Swiftfin/Views/DownloadTaskView/DownloadTaskContentView.swift

```diff
@@ -113,10 +113,10 @@ extension DownloadTaskView {
                 Button {
                     isPresentingVideoPlayerTypeError = false
                 } label: {
-                    Text("Dismiss")
+                    Text(L10n.dismiss)
                 }
             } message: {
-                Text("Downloaded items are only playable through the Swiftfin video player.")
+                Text(L10n.downlodsOnlyPlayableInSwiftfin)
```

Comment:
I actually don't want download-related stuff localized right now, as I still see it very much a work in progress and don't want them publicized.

Reply:
Understood, I'll get those reverted.